tests/CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ ConfigureTest(PREFETCH_TEST prefetch_tests.cpp)
 ConfigureTest(LOGGER_TEST logger_tests.cpp)
 
 # arena MR tests
-ConfigureTest(ARENA_MR_TEST mr/device/arena_mr_tests.cpp GPUS 1 PERCENT 60)
+ConfigureTest(ARENA_MR_TEST mr/device/arena_mr_tests.cpp GPUS 1 PERCENT 100)
```

Comment:
Note: Large allocation occurs here:

https://github.com/rapidsai/rmm/blob/9864b51b2ae83d4bf7f2c934aacebf9cd66ecec2/tests/mr/device/arena_mr_tests.cpp#L487-L495

We should refactor this test to use [`rmm::percent_of_free_device_memory`](https://github.com/rapidsai/rmm/blob/9864b51b2ae83d4bf7f2c934aacebf9cd66ecec2/include/rmm/cuda_device.hpp#L136-L142). I think the only difference is whether it uses `rmm::align_up` vs. `rmm::align_down`, but that seems unimportant here?

Reply:
I'll do this one separately.